docker/controller/README.md

```diff
@@ -0,0 +1,5 @@
+## Docker configuration file for the Turbinia controller
+
+This Docker configuration is for building Turbinia controller image, which can
+be used to access Turbinia logs stored within a Filestore instance as well as
+help troubleshoot issues within the GKE cluster.
```

Comment:
Should we mention something about being able to run turbiniactl or the load tests here too?

Reply:
ack updated this section.